### ISSUE TITLE
Can not receive notification on Android 4.4.2 devices.

### ISSUE THREAD

**Issue by TarunDas005** [evollu/react-native-fcm#613]
I cann't receive push notification on android 4.4.2 devices when i send data in below format.
{
  "to":"/topics/foobar",
  "data": {
    "type":"MEASURE_CHANGE",
    "custom_notification": {
      "body": "Hello",
      "title": "this is title",
      "big_text": "i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large, i am large",
      "color":"#00ACD4",
      "group": "GROUP",
      "priority":"high",
      "show_in_foreground": true
    }
  }
}

but it is working fine on Android 6.0.1

**Comment by evollu** [comment]
it is weird. are you sure it is not device issue?
is this getting called? https://github.com/evollu/react-native-fcm/blob/master/android/src/main/java/com/evollu/react/fcm/MessagingService.java#L25

**Comment by TarunDas005** [comment]
Basically onMessageReceived function not called when i send notification in kitkat devices. already i tested two kitkat devices. but no one can receive notification @evollu

**Comment by evollu** [comment]
then it is firebase config issue. try reproduce with native example?

**Comment by ducpt2** [comment]
I have same issue, it not show banner, only show small icon on top, with version 4.4.2. Any suggest?
Any way, it show the same way (small on top, not show banner) when app in background version 7., Any help? Many thanks.

**Comment by evollu** [comment]
@ducpt2 can you show your payload?

**Comment by ducpt2** [comment]
@evollu Thank you for your quick response. I change code in server like your document, and it work perfectly. But it seem like not elegance way, server need to know who is android, who is ios. Or what i am doing wrong? Any suggest?

**Comment by evollu** [comment]
@ducpt2 yes. it is kind of ugly. I'm open to any suggestions that can improve it.
